azure-pipelines.yml

```diff
@@ -7,14 +7,14 @@ strategy:
   matrix:
     linux-16.04:
       imageName: 'ubuntu-16.04'
-    linux-18.04:
```

Comment:
Why are all elements of the matrix disabled ? Shouldn't there be a separate pipeline for coverage ? Maybe a GitHub workflow would be more appropriate ?